sdk/python/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing/automl-classification-task-bankmarketing.ipynb

```diff
@@ -55,26 +49,28 @@
    "outputs": [],
    "source": [
     "# Import required libraries\n",
-    "import os\n",
     "from azure.identity import DefaultAzureCredential\n",
-    "from azure.ai.ml import MLClient\n",
+    "from azure.identity import AzureCliCredential\n",
+    "from azure.ai.ml import automl, Input, MLClient\n",
     "\n",
     "from azure.ai.ml.constants import AssetTypes\n",
-    "from azure.ai.ml import automl\n",
-    "from azure.ai.ml import Input, command\n",
-    "from azure.ai.ml.entities import Data"
+    "from azure.ai.ml.automl import (\n",
+    "    classification,\n",
+    "    ClassificationPrimaryMetrics,\n",
+    "    ClassificationModels,\n",
+    ")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 1.2. Workspace details\n",
+    "## 1.2. Configure workspace details and get a handle to the workspace\n",
     "\n",
     "To connect to a workspace, we need identifier parameters - a subscription, resource group and workspace name. We will use these details in the `MLClient` from `azure.ai.ml` to get a handle to the required Azure Machine Learning workspace. We use the default [default azure authentication](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.defaultazurecredential?view=azure-python) for this tutorial. Check the [configuration notebook](../../configuration.ipynb) for more details on how to configure credentials and connect to a workspace.\n",
     "\n",
     "By default, we try to use the by default workspace configuration (available out-of-the-box in Compute Instances) or from any Config.json file you might have copied into the folders structure.\n",
```

Comment:
nit: 
1. reorder sentence. **By default** used a couple of times. 
2. Case of `Config.json` need to be modified like below as well?

Reply:
Done